### PR TITLE
Newest readme

### PR DESCRIPTION
**Pull request by rachelzhang0922**
**What this PR does / why we need it**:

**Will this PR make the community happier**? 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: shifu
Comment (tomqin93) [inline shifu:1]: remove submodule import

File: README.md [hunk 1]
```diff
@@ -9,53 +9,51 @@
...
+<div align="center">
 
-|Feature|Description |
-|---|---|
-|🔌 Fast Device integration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Compatible with almost all protocols and drivers.|
-|👨‍💻 Efficient Application development|Shifu structually virtualizes each device and expose its capabilities in the form of APIs.|
-|👨‍🔧 Easy Operation & Maintenance|Kubernetes-native framework, sparing the need for maintaining an additional O&M infrastructure.|
+<h1 style="border-bottom: none">
+<br>
+    Open Source Alternative to GCP IoT Core
```
Comment (tomqin93) [inline README.md:20]: Google IoT or GCP IoT?

File: README.md [hunk 2]
```diff
@@ -9,53 +9,51 @@
 [![golangci-lint](https://github.com/Edgenesis/shifu/actions/workflows/golangci-lint.yml/badge.svg)](https://github.com/Edgenesis/shifu/actions/workflows/golangci-lint.yml)
 
 </div>
-
+
```
Comment (tomqin93) [inline README.md:12]: remove space

File: README.md [hunk 3]
```diff
@@ -65,15 +63,29 @@ We couldn't be more excited to have you.
...
+# Shifu is officially a CNCF landscape project
+
+<div align="center">
+<img width="900px" src="./img/cncf-logo.png"></img>
+</div>
+
+<div align="center">
+<img width="900px" src="./img/cncf.png"></img>
+</div>
+
```
Comment (tomqin93) [inline README.md:89]: put this section before stargazers?

File: README-zh.md [hunk 4]
```diff
@@ -12,63 +12,81 @@
...
+[![Twitter](https://img.shields.io/badge/Twitter-follow-blue)](https://twitter.com/ShifuFramework)
+[![Reddit](https://img.shields.io/badge/Reddit-post-orange)](https://www.reddit.com/r/Shifu/)
+[![GitHub Discussions](https://img.shields.io/badge/GitHub%20Discussions-post-orange)](https://github.com/Edgenesis/shifu/discussions)
 
-[Discord](https://discord.com/channels/1024601454306136074/1039472165399052339) | [Github discussion](https://github.com/Edgenesis/shifu/discussions) | [Twitter](https://twitter.com/ShifuFramework)
+# 开始上手
+欢迎参考🗒️[Shifu技术文档](https://shifu.run/)获取更详细的信息:
+- 🔧[安装Shifu](https://shifu.run/zh-Hans/docs/guides/install/install-shifu-dev)
+- 🔌[设备连接](https://shifu.run/zh-Hans/docs/guides/cases/)
+- 👨‍💻[应用开发](https://shifu.run/zh-Hans/docs/guides/application/)
```
Comment (tomqin93) [inline README-zh.md:71]: replace with `shifu.dev`

File: README.md [hunk 5]
```diff
@@ -65,15 +63,29 @@ We couldn't be more excited to have you.
 [![Reddit](https://img.shields.io/badge/Reddit-post-orange)](https://www.reddit.com/r/Shifu/)
 [![GitHub Discussions](https://img.shields.io/badge/GitHub%20Discussions-post-orange)](https://github.com/Edgenesis/shifu/discussions)
 
-
-# ✍️ Contributing
+# Get Started
+Welcome to check 🗒️[Shifu Documentation](https://shifu.run/) for detailed information on:
+- 🔧[Install](https://shifu.run/docs/guides/install/install-shifu-dev)
+- 🔌[Conenct devices](https://shifu.run/docs/guides/cases/)
+- 👨‍💻[Application Development](https://shifu.run/docs/guides/application/)
```
Comment (tomqin93) [inline README.md:70]: replace with `shifu.dev`

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added some comment, please take a look

File: README.md (hunk 1 above)
Comment (rachelzhang0922) [reply README.md:20]: GCP IoT Core

**Comment by saiyan86** [comment]
@rachelzhang0922 still have conflicts. Please resolve.

**Pull request merged**